src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_config_importer.cljs

```diff
@@ -35,119 +35,120 @@
+   [:div {:style {:fontSize 24 :align "center" :textAlign "center" :paddingBottom "0.5em"}} text]
+   [:hr]])
 
-(defn render-import-modal [config props refs on-close on-import]
-  [comps/Dialog
-   {:width 750
-    :dismiss-self on-close
-    :content (react/create-element
-               [:div {}
-                [:div {:style {:backgroundColor "#fff"
-                               :borderBottom (str "1px solid " (:line-gray style/colors))
-                               :padding "20px 48px 18px"
-                               :fontSize "137%" :fontWeight 400 :lineHeight 1}}
-                 "Import a Method Configuration"
-                 [:hr]
-                 (config "name") [:br]
-                 (config "namespace") [:br]
-                 (config "snapshotId")]
-                 [:div {:style {:position "absolute" :right 2 :top 2}}
-                  [:div {:style {:backgroundColor (:button-blue style/colors) :color "#fff"
-                                 :padding "0.5em" :cursor "pointer"}
-                         :onClick on-close}
-                   (icons/font-icon {:style {:fontSize "60%"}} :x)]]
-                [:div {:style {:padding "22px 48px 40px"
-                               :backgroundColor (:background-gray style/colors)}}
-                 [ModalImportOptionsAndButton {:init-Name (config "name")
-                                               :init-Namespace (config "namespace")
-                                               :init-SnapshotId (config "snapshotId")
-                                               :on-import on-import
-                                               :workspace-id (:workspace-id props)}]]])}])
+; Render out the import form - Need access to the selected configuration, workspace, and import callback.
+(react/defc ConfigurationImportForm
+  {:render
+   (fn [{:keys [props refs]}]
+     (let [selected-config (:selected-method-config props)
+           selected-conf-name (selected-config "name")
+           selected-conf-namespace (selected-config "namespace")
+           selected-conf-snapshot-id (selected-config "snapshotId")
+           workspace-id (:workspace-id props)]
+       [:div {}
+        (formatted-header "Import Method Configuration")
+        ; Render the form fields:
+        (for [[k v] {"Name" selected-conf-name
+                     "Namespace" selected-conf-namespace
+                     "Snapshot Id" selected-conf-snapshot-id
+                     }] (formatted-label-text k v))
+        (for [[k v] {"Destination Name"
+                     (style/create-text-field {:defaultValue selected-conf-name :ref "destinationName"})
+                     "Destination Namespace"
+                     (style/create-text-field {:defaultValue selected-conf-namespace :ref "destinationNamespace"})}]
+          (formatted-label-textfield k v))
+        ;Render the import button and on-click handler for import
+        [comps/Button
+         {:text "Import Configuration ..."
```

Comment:
We generally use ellipses to mean, "I'm going to ask you more questions when you click this." Since this will actually do the import, maybe rename this to simply "Import"